packages/grpc-web/index.d.ts

```diff
@@ -0,0 +1,34 @@
+declare module "grpc-web" {
+
+  export namespace AbstractClientBase {
+    class MethodInfo {
+      constructor (responseType: any,
+                   requestSerializeFn: any,
+                   responseDeserializeFn: any);
+    }
+  }
+
+  export class AbstractClientBase {
+    rpcCall (method: string,
+             request: any,
+             metadata: any,
+             methodInfo: any,
+             callback: (err: any, response: any) => void
+            ): ClientReadableStream;
+  }
+
+  export class ClientReadableStream {
+    on (type: string,
+        callback: (...args: any[]) => any): ClientReadableStream;
+    cancel (): void;
+  }
+
+  export interface GrpcWebClientBaseOptions {
+    format?: string;

```

Comment:
I think that we could add the `suppressCorsPreflight` option as well: `suppressCorsPreflight?: boolean`

Reply:
Will do.